electron-app/src/components/Login.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div v-if="displayed" class="rotki-welcome">
+  <div v-if="displayed" class="rotki-welcome__login">
```

Comment:
If we follow the proper naming scheme for the components this should be just login following bem.